outputs.tf

```diff
@@ -36,4 +36,10 @@ output "datasource_identifier" {
 output "cloudwatch_log_group" {
   value       = length(aws_cloudwatch_log_group.knowledge_base_cwl) > 0 ? aws_cloudwatch_log_group.knowledge_base_cwl[0].name : null
   description = "The name of the CloudWatch log group for the knowledge base.  If no log group was requested, value will be null"
-}
\ No newline at end of file
+}
+
+output "bedrock_agent" {
```

Comment:
Do we want to reduce this to some specific outputs than the entire object ?

Reply:
Because it is unknown what callers may want to inspect, I believe the entire Amazon Bedrock Agent should be returned.